src/main/java/com/travelBill/config/ApplicationConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package com.travelBill.config;
+
+import static org.springframework.util.StringUtils.isEmpty;
```

Comment:
You should check env vars before the app starts and inform in case of fail. No need to run the app and wait when e.g. getTelegramApiKey be invoked.